Formula/wxmac.rb

```diff
@@ -1,15 +1,14 @@
 class Wxmac < Formula
   desc "Cross-platform C++ GUI toolkit (wxWidgets for macOS)"
   homepage "https://www.wxwidgets.org"
-  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2"
-  sha256 "440f6e73cf5afb2cbf9af10cec8da6cdd3d3998d527598a53db87099524ac807"
+  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.5/wxWidgets-3.1.5.tar.bz2"
+  sha256 "d7b3666de33aa5c10ea41bb9405c40326e1aeb74ee725bb88f90f1d50270a224"
   license "wxWindows"
-  revision 2
   head "https://github.com/wxWidgets/wxWidgets.git"
 
   livecheck do
     url :stable
-    regex(/^v?(\d+\.\d*[02468](?:\.\d+)*)$/i)
+    regex(/^v?(\d+(?:\.\d+)*)$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

This was using `*` because the previous regex had more than one version part before the repeating `(?:\.\d+)` part. This should simply use the standard regex for Git tags like `v1.2.3` (which uses `+`) at this point.